### ISSUE TITLE
Difficult to tell whether I am in Dev or Stable

### ISSUE THREAD

**Issue by aaronchantrill** [NaomiProject/naomi-website#15]
I have no idea how this was implemented or how much of a pain it would be to fix it, but I would like the dropdown control for version to change depending on whether I am on the stable or dev documentation. Right now, the only way I can tell if I am on the Dev documentation is by looking for the "/dev/" in the url. It would be lovely if the currently selected option would be visible in the "rolled up" state and would be bold in the dropdown with the non-selected option looking more like a traditional link.

Currently, this is how the selector looks whether you are in dev or stable:
![stable](https://user-images.githubusercontent.com/7544675/51916887-432ed300-23ac-11e9-8ba3-d04522e53e61.png)

and here's how I would like it to look in dev:
![dev](https://user-images.githubusercontent.com/7544675/51916886-432ed300-23ac-11e9-8659-cabbfda1b62e.png)

There is also currently no way to return from dev to stable without editing the url directly, and if you select dev again, you are sent to a page with two "/dev/"'s in the url, which returns a 404 error.

**Comment by AustinCasteel** [comment]
This is an issue with the site, not the documentation. When we made the change from the dropdown being latest and then past version below to just stable and dev the functionality broke. I have been trying to figure out why for some time now. I will move it to the top of my list of things to do and get it fixed as soon as I can.

**Comment by redragonx** [comment]
I had some time to look at the problem today with @AustinCasteel. I narrowed it down to some code that selects the current "version" based on the main URL.

```
    if (this.$site.base && this.$site.base.indexOf('v') > 0) {
      this.currentVersion = this.$site.base.replace('v', '').replace(/\//g, '')
```

I believe Austin will be pushing a fix soon.

**Comment by AustinCasteel** [comment]
Fixed in [PR #26](https://github.com/NaomiProject/naomi-website/pull/26). Thank you both for your help in the matter!
